doc/mgr/dashboard.rst

```diff
@@ -619,6 +619,12 @@ We provide a set of CLI commands to manage user accounts:
 
   $ ceph dashboard ac-user-set-password <username> <password>
 
+- *Change Password Hash*::
```

Comment:
Can you please add a small note saying that this command might be useful for importing an external database of users?